recipes/antismash/meta.yaml

```diff
@@ -1,64 +1,44 @@
 {% set name = "antiSMASH" %}
-{% set version = "4.2.0" %}
-{% set sha256 = "cf284c2a18ee8d76762a599b6c6d6cadf5a0adefe2078c84875b22c96dd777b2" %}
+{% set version = "5.1.1" %}
+{% set sha256 = "7e59a92a01e315dfb9d28d734d733a233dc5366eb0245fb8f91d9b07d4c7af82" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 build:
+  number: 0
   noarch: python
-  number: 2
 
 source:
-  url: https://dl.secondarymetabolites.org/releases/{{ version }}/antismash-{{ version }}.tar.gz
+  url: https://github.com/antismash/antismash/archive/{{version.replace(".", "-")}}.tar.gz
```

Comment:
I'd prefer if you would continue to get the releases from the download server. That's the tarball we use for all of our other installs, and while they of course should be in sync, I'd like to avoid the "surprise mechanics" in support requests if they should ever be out of sync.